app/javascript/api/adapters/listing/UnitSummariesAdapter.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  UnitsSummarized,
+  UnitSummary,
+  MinMaxCurrency,
+  MinMax,
+} from "@bloom-housing/backend-core/types"
+
+import RailsRentalListing from "../../types/rails/listings/RailsRentalListing"
+import RailsRentalUnitSummary from "../../types/rails/listings/RailsRentalUnitSummary"
+import { Adapter } from "../adapter"
+
+// todo: implement with correct formatting
+const asCurrencyString = (value: number): string => `$${value}`
+
+const getMinMaxCurrency = (min: number, max: number): MinMaxCurrency => ({
+  min: min != null && asCurrencyString(min),
+  max: max != null && asCurrencyString(max),
+})
+
+const getMinMax = (min: number, max: number): MinMax => ({ min, max })
+
+const UnitSummaryAdapter: Adapter<RailsRentalUnitSummary, UnitSummary> = (
+  s: RailsRentalUnitSummary
+) => ({
+  unitType: s.unitType,
+  minIncomeRange: getMinMaxCurrency(s.absoluteMinIncome, s.absoluteMaxIncome),
```

Comment:
So this is called the min income range, but for us it's actually the full income range. This might work for now, but we'll have to come back to it with bloom